packages/playground-ui/theme.css

```
@@ -395,6 +395,42 @@ html.light {
   --shadow-glow-accent2: 0 0 20px oklch(0.637 0.237 25.331 / 0.25);
   --shadow-focus-ring: 0 0 0 3px oklch(0.723 0.219 149.579 / 0.12);
 
+  --animate-signals-connector: signals-connector 2.8s ease-in-out infinite;
```

Stylelint (17.14.0)

[error] Expected no empty line before custom property (custom-property-empty-line-before)

(custom-property-empty-line-before)